docs/src/components/ConsentManager.js

```diff
@@ -1,9 +1,36 @@
 import React from 'react'
-import { ConsentManager, openConsentManager } from '@segment/consent-manager'
-import inEU from '@segment/in-eu'
+import inRegions from '@segment/in-regions'
+import { ConsentManager, CloseBehavior } from '@segment/consent-manager'
 import segmentWriteKey from '../../segmentWriteKey'
 
 export default () => {
+  const inCA = inRegions(['CA'])
+  const inEU = inRegions(['EU'])
+  const shouldRequireConsent = inRegions(['CA', 'EU'])
```

Comment:
Non-blocking: maybe `inRegions` should export a list of valid regions rather than relying on magic strings.

Reply:
Totally! I'd like to do that in a next version, maybe in 2020 😉